apps/test/shareWarningsDialogTest.js

```diff
@@ -0,0 +1,225 @@
+window.React = require('react');
+require('react/addons');
+var ReactTestUtils = React.addons.TestUtils;
+
+var testUtils = require('./util/testUtils');
+var assert = testUtils.assert;
+testUtils.setupLocales('Applab');
+var ShareWarningsDialog = require('@cdo/apps/templates/ShareWarningsDialog.jsx');
+var ShareWarnings = require('@cdo/apps/templates/ShareWarnings.jsx');
+var AgeDropdown = require('@cdo/apps/templates/AgeDropdown.jsx');
+
+var msg = require('@cdo/apps/locale');
+
+describe('ShareWarningsDialog', function () {
+  function isTagWithText(childComponent, tagName, text) {
+    if (!ReactTestUtils.isDOMComponent(childComponent)) {
+      return false;
+    }
+    var dom = childComponent.getDOMNode();
+    return dom.tagName === tagName && dom.innerHTML === text;
+  }
+
+  function isDataPromptDiv(childComponent) {
+    return isTagWithText(childComponent, 'DIV', msg.shareWarningsStoreData());
+  }
+
+  function isAgeDiv(childComponent) {
+    return isTagWithText(childComponent, 'DIV', msg.shareWarningsAge());
+  }
+
+  it('only shows age prompt if not is13Plus and we dont storeData', function () {
```

Comment:
Nit: I got confused reading "we dont storeData" because I thought it was talking about the local storage, and because there's no storeData variable in the code. How about "the app doesnt store data"?